templates/sysdig-cloud/0/rancher-compose.yml

```diff
@@ -0,0 +1,28 @@
+.catalog:
+  name: "Sysdig Cloud"
+  version: "v0"
+  description: "Container-Native Application and Infrastructure Monitoring"
+  uuid: sysdig-cloud-0
+  questions:
+    - variable: "SDC_ACCESS_KEY"
+      label: "Sysdig Cloud access key"
+      description: "Your unique Sysdig Cloud access key - register for a Sysdig Cloud account at www.sysdig.com to receive a key."
+      type: "string"
+      required: true
+    - variable: "SDC_TAGS"
+      label: "Sysdig Cloud tags"
+      description: "Tags to be applied to all hosts on which the Sysdig Cloud container is deployed - these will surface in the Sysdig Cloud app. Should be a comma-separated list in the form of TAG_NAME:TAG_VALUE. For example: role:webserver,location:europe."
+      type: "string"
+      required: false
+    - variable: "VERSION"
+      label: "Sysdig Cloud version"
+      description: "Optionally specify a version of the Sysdig Cloud container to pull (default will pull latest stable version)."
+      type: "string"
+      default: "latest" 
+      required: false
```

Comment:
Needs to be required, if its blank the template will fail to launch because of the ':'